ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,65 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm
+using ITensorNetworks:
+  ITensorNetworks,
+  ITensorNetwork,
+  vertextype,
+  vertex_data,
+  externalinds,
+  contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function to_einexpr(tn::AbstractITensorNetwork)
+  IndexType = Any
+
+  tensor_exprs = EinExpr{IndexType}[]
+  inds_dims = Dict{IndexType,Int}()
+
+  for v in vertices(tn)
+    tensor_v = tn[v]
+    inds_v = collect(inds(tensor_v))
+    push!(tensor_exprs, EinExpr{IndexType}(; head=inds_v))
+    merge!(inds_dims, Dict(inds_v .=> size(tensor_v)))
+  end
+
+  externalinds_tn = collect(externalinds(tn))
+  return SizedEinExpr(sum(tensor_exprs; skip=externalinds_tn), inds_dims)
+end
+
+function tensor_inds_to_vertex(tn::AbstractITensorNetwork)
+  mapping = Dict{Set{IndexType},VertexType}()
+
+  for v in vertices(tn)
+    tensor_v = tn[v]
+    inds_v = collect(inds(tensor_v))
+    mapping[Set(inds_v)] = v
+  end
+
+  return mapping
+end
+
+function EinExprs.einexpr(tn::ITensorNetwork; optimizer::EinExprs.Optimizer)
+  expr = to_einexpr(tn)
+  return einexpr(optimizer, expr)
+end
```

Comment:
Is this function necessary? Or can we just combine it directly with `einexpr()` above?

Reply:
Now that we have `to_einexpr`, I think it's no longer needed.